dev_cmds/utils/paths.ts

```diff
@@ -6,7 +6,7 @@ const packagesDir = resolve(repoRoot, "packages");
 const devcmdCliPackageDir = resolve(packagesDir, "devcmd-cli");
 const devcmdPackageDir = resolve(packagesDir, "devcmd");
 
-const devPath = path.join(repoRoot, 'dev');
+const devPath = path.join(repoRoot, "dev");
```

Comment:
the `path` import ([this comment](https://github.com/XITASO/devcmd/pull/26#discussion_r674070699)) and naming ([this comment](https://github.com/XITASO/devcmd/pull/26#discussion_r674070272)) is still open